python/eos/analysis.py

```diff
@@ -155,6 +155,18 @@ def __init__(self, priors, likelihood, global_options={}, manual_constraints={},
         for n in varied_parameter_names - used_parameter_names:
             eos.warn('likelihood does not depend on parameter \'{}\'; remove from prior or check options!'.format(n))
 
+    def __sanitize_manual_input(self, data):
+        """Helper function that cast to float all entries of a manual_constraint"""
+        if type(data) is dict:
```

Comment:
Maybe separate with two brief comments the recursion and casting sections.

Reply:
The comment is also incomplete. You do more than cast to ``float``. You also convert iterables to lists.

Reply:
That is true...